### PR TITLE
fix: access to download binary from mirror sites

### PR DESCRIPTION
**Pull request by ShangguanQuail**
Installation will fail when network is not access to https://yt-dl.org/downloads/latest/youtube-dl

### CODE REVIEW

File: lib/downloader.js [hunk 1]
```diff
@@ -13,7 +13,9 @@ const isWin = flags.includes('--platform=windows') || require('./util').isWin
 let dir, filePath
 const defaultBin = path.join(__dirname, '..', 'bin')
 const defaultPath = path.join(defaultBin, 'details')
-const url = 'https://yt-dl.org/downloads/latest/youtube-dl'
+const url = process.env.YOUTUBE_DL_BINARY_MIRROR 
+|| process.env.youtube_dl_binary_mirror
```
Comment (Kikobeats) [inline lib/downloader.js]: Can you rename it to `YOUTUBE_DL_DOWNLOAD_HOST`? 🙂

Just to be similar to https://github.com/puppeteer/puppeteer/blob/eddb23b52175a3dad8cec4a5e9b3c13e58f6f144/docs/api.md

**Comment by Kikobeats** [comment]
Thanks for that, additionally can you add it as part of the README?

Similar to https://github.com/puppeteer/puppeteer/blob/eddb23b52175a3dad8cec4a5e9b3c13e58f6f144/docs/api.md

**Comment by ShangguanQuail** [comment]
> Thanks for that, additionally can you add it as part of the README?
> 
> Similar to https://github.com/puppeteer/puppeteer/blob/eddb23b52175a3dad8cec4a5e9b3c13e58f6f144/docs/api.md

yeah，it looks like a role model.

**Comment by Kikobeats** [comment]
Released a `2.2.0` 🎉

**Pull request merged**